values.yaml

```diff
@@ -132,15 +135,15 @@ postgresql:
 # **Zabbix Proxy** configurations
 zabbixproxy:
   # -- Enables use of **Zabbix Proxy**
-  enabled: true
+  enabled: false
   # -- Number of replicas of ``zabbixproxy`` module
   replicaCount: 1
   resources: {}
   image:
     # -- Zabbix proxy Docker image name
     repository: zabbix/zabbix-proxy-sqlite3
     # -- Tag of Docker image of Zabbix proxy
-    tag: ubuntu-6.0.4
+    #tag: ubuntu-6.0.5
```

Comment:
Hi @sa-ChristianAnton!

With tag centralized, this line can be removed, correct?

Reply:
commit 3f529c7